examples/nextjs-todo/components/ui/TodoInput.tsx

```diff
@@ -0,0 +1,145 @@
+import React, { useState, useEffect } from "react";
+import { TBaseParameter, TUnit } from "@arkejs/client";
+import { Form, FormField } from "@arkejs/form";
+import useClient from "@/arke/useClient";
+import { Button, Input } from "@arkejs/ui";
+
+type ID = string;
+export interface CrudState {
+  add?: boolean;
+  edit?: boolean | ID;
+  delete?: boolean | ID;
+}
+
+export default function TodoInput({ onSetModal, onModal, onRefreshPage }: any) {
```

Comment:
Make props more readable. Change the component name into `TodoForm` and use the `Dialog` component in it.

- `onSetModal` should be `onClose` 
- `onModal` should be `onSubmit`
- `onRefreshPage` is probably useless